tests/multisigNFTs_test.go

```diff
@@ -0,0 +1,215 @@
+		{"maintenance-cast-action", false, false, "(Approve)-Cast action to approve carlo as non fungible token issuer address, Happy path. commit", "nft-maintainer-1", "0cin", 0, CastAction{"nft-maintainer-1", "approve", 2}, "", nil},
+		{"maintenance-cast-action", false, false, "(Approve)-Cast action to approve carlo as non fungible token issuer address, Happy path. commit", "nft-maintainer-3", "0cin", 0, CastAction{"nft-maintainer-3", "approve", 2}, "", nil},
+
+		//add nft-jeansoon as nonfungible provider address
+		{"maintenance", false, false, "3. Proposal, add nonfungible provider address, Happy path. commit", "nft-maintainer-2", "0cin", 0, MaintenanceInfo{"add", "Add non fungible fee provider address", "Add jeansoon as non fungible provider address", "nonFungible", "", "", "nft-jeansoon", FeeCollector{}, "nft-maintainer-2", ""}, "", nil},
+		{"maintenance-cast-action", false, false, "Cast action to approve jeansoon as non fungible token provider address, Happy path. commit", "nft-maintainer-1", "0cin", 0, CastAction{"nft-maintainer-1", "approve", 3}, "", nil},
+		{"maintenance-cast-action", false, false, "Cast action to approve jeansoon as non fungible token provider address, Happy path. commit", "nft-maintainer-3", "0cin", 0, CastAction{"nft-maintainer-3", "approve", 3}, "", nil},
+
+		//add nameservice fee collector with maintenance. (multisig-acc-1 is whitelisted.)
+		{"maintenance", false, false, "4. Proposal, add token fee collector address, Happy path. commit", "nft-maintainer-2", "0cin", 0, MaintenanceInfo{"add", "Add token fee collector", "Add mostafa as nameservice fee collector", "fee", "", "", "", FeeCollector{Module: "nonFungible", Address: "multisig-acc-1"}, "nft-maintainer-2", ""}, "", nil},
+		{"maintenance-cast-action", false, false, "(Approve)-Cast action to approve mostafa as nameservice fee collector, Happy path. commit", "nft-maintainer-1", "0cin", 0, CastAction{"nft-maintainer-1", "approve", 4}, "", nil},
+		{"maintenance-cast-action", false, false, "(Approve)-Cast action to approve mostafa as nameservice fee collector, Happy path. commit", "nft-maintainer-3", "0cin", 0, CastAction{"nft-maintainer-3", "approve", 4}, "", nil},
+
+		//4. module : fee_test
+		{"fee", false, false, "assign zero-fee to mostafa-commit", "nft-fee-auth", "0cin", 0, feeInfo{"assign-acc", "zero", "nft-mostafa", "", "", "", "", "nft-fee-auth"}, "", nil},
+
+		// NFTs testing :
+		{"multiSig", false, false, "Create MultiSig Tx for NFTs [Create-token] - submit counter+0 - Happy Path commit.", "multisig-acc-1", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "grp-addr-1", 0, internalTx3_createToken_00}, "MEMO : xxxxx", nil},
+		{"multiSig", false, false, "Create MultiSig Tx for NFTs [Approve-token] - Happy path -  commit.", "multisig-acc-1", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "grp-addr-1", 1, internalTx3_approveToken_00}, "MEMO : xxxxx", nil},
```

Comment:
We have problem we have.
Approve token is not a multi-sig message. Authorizers can approve or reject the token.
I think we have serious problem on testing NFT-multisig. Imagine a multi-sig account create a NF Token. Then it takes 3 second to broadcast the internal TX (which is create_nft_token). The next line which is approve token will be failed.

Reply:
Hi, mostafa

Need your guideline on these two items. 

As I knew, normal Single-sign for NFTs need to go through APPROVE process due to the 'Internal approval-flag'. For multiSig part, need a discuss on this.